.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+---
+repos:
+  - repo: https://github.com/ccoVeille/mockery
```

Comment:
Is this just in here for testing right now?  Unless mockery uses itself to generate mocks, and wants to use the pre-commit functionality, I don't think this is required to get this to work for consumers.

Reply:
Yes, only for the test. I forgot to mention about it in the PR

I had to create a fake "2.43.3" tag to be able to make it work.